config/cobbler/distro_signatures.json

```diff
@@ -3245,7 +3245,7 @@
         "kernel_options_post": "",
         "boot_files": []
       },
-      "XP": {
+      "xp": {
```

Comment:
Just out of curiosity: Why this change?

Reply:
Now, when validating, `os_version` from `Item` is always converted to lower case:
https://github.com/cobbler/cobbler/blob/40110c5fa48f2130a276df1a0a9f42ffff7ebe59/cobbler/validate.py#L278-L280    
Therefore, if both the `Item` and `SIGNATURE_CACHE` contain "XP", then an error occurs:
```
[2023-01-02_142247_Loading items_62893df81a524ce58846911623f88a76] 2023-01-02T14:22:47 - INFO | Exception value: os_version for breed "windows" must be one of 2003, 2008, 2012, 2016, 2019, XP, 7, 8, 10, 11, given was "xp"
[2023-01-02_142247_Loading items_62893df81a524ce58846911623f88a76] 2023-01-02T14:22:47 - INFO | Exception Info:
```
When comparing, either both values must be converted to lower case, or upper case values must not be allowed in `distro_signatures.json`.